src/components/Contentful/SectionWithBackgroundClassic.vue

```diff
@@ -0,0 +1,167 @@
+	},
+	props: {
+		/**
+		 * Content group content for background from Contentful type Background
+		* */
+		backgroundContent: {
+			type: Object,
+			default: () => {},
+		},
+		verticalPadding: {
+			type: Object,
+			default: () => {},
+		}
+	},
+	data() {
+		return {
+			sourceSizes: [
+				{
+					width: 1920,
+					height: 650,
+					media: 'min-width: 1441px',
+				},
+				{
+					width: 1440,
+					height: 620,
+					media: 'min-width: 1025px',
+				},
+				{
+					width: 1024,
+					height: 441,
+					media: 'min-width: 681px',
+				},
+				{
+					width: 680,
+					height: 372,
+					media: 'min-width: 481px',
+				},
+				{
+					width: 480,
+					height: 540,
+					media: 'min-width: 0px',
+				},
+			],
+		};
+	},
+	computed: {
+		backgroundStyle() {
+			if (this.backgroundContent?.backgroundColor) {
+				return {
+					'background-color': this.backgroundContent?.backgroundColor
+				};
+			}
+			return {};
+		},
+		isBackgroundVideo() {
+			return this.backgroundContent?.backgroundMedia?.file?.contentType.includes('video');
+		},
+		isBackgroundImage() {
+			return this.backgroundContent?.backgroundMedia?.file?.contentType.includes('image');
+		},
+		backgroundMedia() {
+			return {
+				description: this.backgroundContent?.backgroundMedia?.description ?? '',
+				title: this.backgroundContent?.backgroundMedia?.title ?? '',
+				url: this.backgroundContent?.backgroundMedia?.file?.url ?? ''
+			};
+		},
+		verticalPaddingClasses() {
+			// Defaults
+			const defaultSmClasses = 'tw-pt-2 tw-pb-2';
+			const defaultLgClasses = 'lg:tw-pt-4 lg:tw-pb-4';
+			let classes = '';
+
+			// apply small or default classes
+			if (this.verticalPadding && this.verticalPadding.sm) {
+				// eslint-disable-next-line max-len
+				classes = `${classes} tw-pt-${this.verticalPadding.sm.top} tw-pb-${this.verticalPadding.sm.bottom}`;
+			} else {
+				classes = `${defaultSmClasses}`;
+			}
+
+			// apply medium classes
+			if (this.verticalPadding && this.verticalPadding.md) {
+				// eslint-disable-next-line max-len
+				classes = `${classes} md:tw-pt-${this.verticalPadding.md.top} md:tw-pb-${this.verticalPadding.md.bottom}`;
+			}
+
+			// apply large classes or default classes
+			if (this.verticalPadding && this.verticalPadding.lg) {
+				// eslint-disable-next-line max-len
+				classes = `${classes} lg:tw-pt-${this.verticalPadding.lg.top} lg:tw-pb-${this.verticalPadding.lg.bottom}`;
+			} else {
+				classes = `${classes} ${defaultLgClasses}`;
+			}
+
+			// apply xl classes
+			if (this.verticalPadding && this.verticalPadding.xl) {
+				// eslint-disable-next-line max-len
+				classes = `${classes} xl:tw-pt-${this.verticalPadding.xl.top} xl:tw-pb-${this.verticalPadding.xl.bottom}`;
+			}
+
+			// return final class declarations
+			return `${classes}`;
+		}
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.section-with-background {
+	&__media {
+		// over ride kvContentfulImg nested img styling
+		::v-deep img {
```

Comment:
@ryan-ludwig @eddieferrer any thoughts on if this is really needed or a way to do it with tw?

Reply:
Maybe add a prop to kvcontentfulimg which would then add the appropriate classes

Reply:
Yea I think that's a good idea.

Reply:
I would try using KvContentfulImg from kv-components. That component has some modifications, namely the img tag deep in the component has `object-fit; inherit` which may solve your problem?

Reply:
Oh good call, will check it out with the new img component first.

Reply:
Switched to new kv-component version which should work fine with some small updates made in that repo.